pep_sphinx_extensions/pep_processor/html/pep_html_redirects.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from pathlib import Path
```

Comment:
```suggestion
from pathlib import Path
from typing import TYPE_CHECKING
```

Both `pathlib` and `typing` are stdlib modules, so should go in the same block per PEP 8, and be sorted lexicographically.